atm.js

```diff
@@ -1,3 +1,70 @@
 $(document).ready(function(){
-// Uh oh -- it's saying `$` is undefined! Something's missing from `index.html`...
-});
+
+var type= prompt("For saving account enter 1 or for saving account enter 2, please")
+if(type==1)
+{
+  var depositButton = $("#checking .deposit")
+  var withdrawButton =$("#checking .withdraw")
+  var balance1 =$("#checking .balance")
+  var balance =balance1.html().replace("$","")
+  var amount1 = $("#checking .input");
+}
+else if (type==2){
```

Comment:
Any thoughts on how you would implement this without having to use a `prompt` to determine which account is being interacted with?